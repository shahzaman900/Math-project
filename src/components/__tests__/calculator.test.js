import { render, screen } from '@testing-library/react';
import Calculator from '../calculator';

test('Testing Calculator component', () => {
  render(<Calculator />);
  const linkElement = screen.getByText(/AC/i);
  expect(linkElement).toBeInTheDocument();
});
