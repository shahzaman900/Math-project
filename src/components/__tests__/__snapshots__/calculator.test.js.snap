// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<div className="calculator">
<div className="display">


  </div>
  <div
    className="row"
  >
    <button
      onClick={[Function]}
    >
      AC
    </button>
    <button
      onClick={[Function]}
    >
      +/-
    </button>
    <button
      onClick={[Function]}
    >
      %
    </button>
    <button
      className="yellow"
      onClick={[Function]}
    >
      รท
    </button>
  </div>
  <div
    className="row"
  >
    <button
      onClick={[Function]}
    >
      7
    </button>
    <button
      onClick={[Function]}
    >
      8
    </button>
    <button
      onClick={[Function]}
    >
      9
    </button>
    <button
      className="yellow"
      onClick={[Function]}
    >
      x
    </button>
  </div>
  <div
    className="row"
  >
    <button
      onClick={[Function]}
    >
      4
    </button>
    <button
      onClick={[Function]}
    >
      5
    </button>
    <button
      onClick={[Function]}
    >
      6
    </button>
    <button
      className="yellow"
      onClick={[Function]}
    >
      -
    </button>
  </div>
  <div
    className="row"
  >
    <button
      onClick={[Function]}
    >
      1
    </button>
    <button
      onClick={[Function]}
    >
      2
    </button>
    <button
      onClick={[Function]}
    >
      3
    </button>
    <button
      className="yellow"
      onClick={[Function]}
    >
      +
    </button>
  </div>
  <div
    className="row"
  >
    <button
      className="zero"
      onClick={[Function]}
    >
      0
    </button>
    <button
      onClick={[Function]}
    >
      .
    </button>
    <button
      className="yellow"
      onClick={[Function]}
    >
      =
    </button>
  </div>
</div>
`;
